client/app/components/packages/pas-form/show.js

```diff
@@ -0,0 +1,53 @@
+import Component from '@glimmer/component';
+
+export default class PackagesPasFormShowComponent extends Component {
+  get proposedDevelopmentTypes() {
+    const { pasForm } = this.args.package;
+    const arr = [];
+
+    if (pasForm.dcpProposeddevelopmentsitenewconstruction) {
```

Comment:
Hmm, this style seems a more verbose, hybrid of the way you had previously and the way we discussed during engie standup. My personal preference is one of the more concise versions.
Was there something about one of those styles that didn't work?